tests/components/griddy/test_config_flow.py

```diff
@@ -0,0 +1,53 @@
+"""Test the Griddy Power config flow."""
+from asynctest import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.griddy.config_flow import CannotConnect
+from homeassistant.components.griddy.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.griddy.config_flow.GriddyGateway.async_verify_connection",
```

Comment:
We want to patch the library code not our config flow code. Ie: `"homeassistant.components.griddy.config_flow.AsyncGriddy.async_getnow"`.

Reply:
@MartinHjelmare Will change.

Also does ```script/scaffold/templates/config_flow/tests/test_config_flow.py``` need to be adjusted as this is copy pasta from there?

Reply:
Yes, it should.